csrc/scheduler/utils.cpp

```diff
@@ -575,7 +575,7 @@ TvProperties getProperties(
   // Start from the inner most dimension, and work outwards. If this is a 3D
   // pattern, i.e. theres a pattern like [r0, r1, i2, r3] or [i0, r1, r2, i3,
   // i4] then compute the inner most dimension to compute separately.
-  const auto& root_dom = tv->getMaybeRFactorDomain();
+  const auto& root_dom = tv->getRootDomain();
```

Comment:
I think using root domain is more aligned with the deleted comment:
```C++
  // This is backwards from how we normally think about grabbing root dimensions
  // to process. If we're in a reduction scheduler and we're using the rfactored
  // reduction tensor view, we don't care about the rfactor domain, we care
  // about the root domain because we're looking to vectorize the reads (input
  // tensor views). Otherwise we do want the rfactor domain. So this is the
  // reverse of our typical check, we actually want to selectively ignore the
  // rfactor domain.
```

Reply:
IIUC, at this point, there should be no reduction rfactor, right? Should we assert that?

Reply:
Yes, there should be no reduction rfactor, but there could be transpose rfactor fused in the reduction, for the case of matmul. So I would like to leave the door open for having an rFactor.

Reply:
But transpose tensors should not have reduction domains, right?

Reply:
They can, for such a case, it is a fused transpose-reduction. An example for transpose-reduction is NN matmul, where the reduction tensor has root domain `[N, rK, M]` but rFactor domain `[M, N, rK]`

Reply:
OK, I forgot that's now possible. Makes sense.